### PR TITLE
Updated READMEs per the new container model we're using.

### PR DESCRIPTION
**Pull request by yi2020**

### CODE REVIEW

**Comment by yi2020** [comment]
@ivantikal please review the instructions for how to use the container, including the function in the .rc file, to verify that it aligns with the new container that will be made available in prod soon.

**Comment by yi2020** [comment]
@charlesk92 appreciate your review of the READMEs, let me know if there are any changes needed.

File: README.md [hunk 1]
```diff
@@ -23,25 +23,42 @@ Cloudrail currently supports Terraform files used with the AWS cloud provider.
 
 ## Requirements
 - Python >= 3.8
```
Comment (ivantikal) [inline README.md:25]: no Python requirement anymore

File: README.md [hunk 2]
```diff
@@ -23,25 +23,42 @@ Cloudrail currently supports Terraform files used with the AWS cloud provider.
...
 ```
-~ # pip3 install cloudrail --upgrade --extra-index-url https://indeni.jfrog.io/indeni/api/pypi/cloudrail-cli-pypi/simple
+docker run --rm -it -v $PWD:/data -v cloudrail:/indeni indeni/cloudrail-cli --version
 ```
 
+When running on your own workstation, we recommend adding this function to your shell's .rc file (~/.bashrc, ~/.zshrc, etc.):
+```shell script
+cloudrail () {
+  printf 'Checking for an updated cloudrail image (may take a few minutes if a new one is downloaded)...'
+  docker pull indeni/cloudrail-cli > /dev/null printf '\r \n'
```
Comment (ivantikal) [inline README.md:55]: Remove "printf '\r \n'"  - this not correct to have in this and also this 'printf' is not really necessary (I just wanted to clean the previous message)

**Comment by yi2020** [comment]
I actually agree with your printf solution (to remove the previous message
and keep things clean). However, I forgot to add a newline before the
printf, so added now.

File: README.md [hunk 3]
```diff
@@ -22,26 +22,44 @@ Cloudrail's main advantages vs existing tools are:
...
-~ # pip3 install cloudrail --upgrade --extra-index-url https://indeni.jfrog.io/indeni/api/pypi/cloudrail-cli-pypi/simple
+docker run --rm -it -v $PWD:/data -v cloudrail:/indeni indeni/cloudrail-cli --version
 ```
 
+When running on your own workstation, we recommend adding this function to your shell's .rc file (~/.bashrc, ~/.zshrc, etc.):
+```shell script
+cloudrail () {
+  printf 'Checking for an updated cloudrail image (may take a few minutes if a new one is downloaded)...'
+  docker pull indeni/cloudrail-cli > /dev/null
+  printf '\r \n'
```
Comment (ivantikal) [inline README.md:56]: it should be:
printf '\r                                                                  \n'

Comment (yi2020) [reply README.md]: How about now?

**Review by charlesk92 — APPROVED** [review]

**Pull request merged**